Samples/WinMLSamplesGallery/WinMLSamplesGallery/Common/SamplesGrid.xaml

```diff
@@ -0,0 +1,49 @@
+<Page
+    x:Class="WinMLSamplesGallery.SamplesGrid"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinMLSamplesGallery"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Page.Resources>
+        <DataTemplate x:Name="ImageOverlayTemplate" x:DataType="local:Sample">
+            <Grid Height="150" Width="300" Background="#e6e6e6" BorderThickness="4" Padding="10,5,10,1">
```

Comment:
what is the resize behavior when the height and width are fixed like this? does it look right when you resize smaller than this?

Reply:
Resize seems fine since other parts of the page shrink then the cards go from being horizontally stacked to vertically stacked at smaller widths. You can check by shrinking the application window from the main page.